pkg/asset/installconfig/ovirt/validaton.go

```diff
@@ -125,3 +127,21 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+func validURL(val interface{}) error {
+	if urlStr, ok := val.(string); ok {
+		url, err := url.ParseRequestURI(urlStr)
+
+		if err != nil {
+			return errors.Errorf("The specified URL is invalid, got %s", urlStr)
+		}
+
+		if url.Scheme != "https" {
+			return errors.Errorf("The only URL scheme accepted is https, but got %s", url.Scheme)
+		}
+
```

Comment:
Tiny comment: please remove extra space.

Reply:
Hi @dougsland which extra spaces do you mean?
I checked the format with `go fmt` and seems ok.

Reply:
Think I made it